src/main/java/seedu/address/logic/commands/FindCommand.java

```diff
@@ -53,7 +53,7 @@ public class FindCommand extends Command {
      *                         a predicate which returns true only when all the keywords match, is loaded.
      */
     public FindCommand(ContainsSpecificKeywordsPredicate specificPredicate) {
-        requireNonNull(specificPredicate);
+        assert specificPredicate != null;
```

Comment:
Ok, simple but important assertion

Reply:
Technically the `requireNonNull` function already enforces that the value should be not null... and it also runs at runtime too

Reply:
> Technically the `requireNonNull` function already enforces that the value should be not null... and it also runs at runtime too

Touche. I did think about this too lol. I'll try find somewhere else that i can put assertions instead.